ios/Sources/Model/MultiLangText.swift

```diff
@@ -0,0 +1,32 @@
+import DroidKaigiMPP
+
+public struct MultiLangText: Equatable {
+    public var enTitle: String
+    public var jaTitle: String
+    public var currentLangTitle: String
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)
Why we need `currentLangTitle` here as a stored property?
In my opinion, to reflect the change of `Lang`, a `View` using `MultiLangText` can observe the change of `Lang` directly without `currentLangTitle` and get the proper value from `getByLang(lang:)`.

Reply:
Basically, the model converted from KMM model has equal properties.
But I agree your opinion, so I will remove this property.
If there is any inconvenience later, I will revert it too. 🙏🏼